gui/credits.h

```diff
@@ -461,8 +434,6 @@ static const char *credits[] = {
 "C2""(Pentagram)",
 "C0""Dominik Reichardt",
 "C2""(Pentagram)",
-"C0""Matthew Jimenez",
-"C2""(Pentagram)",
```

Comment:
an author is being removed from here. is this expected? I modified the tool to not write any ascii strings.

Reply:
It's expected - the relevant commit is this one: 9c4e6244100f89b0e078006de7ba73dbb36d46a4.

Reply:
Ah, I see. thanks!